opencensus/stats/exporters/prometheus_exporter.py

```diff
@@ -0,0 +1,339 @@
+# Copyright 2018, OpenCensus Authors
```

Comment:
I commented about keeping this file name as `prometheus.py` instead of `prometheus_exporter.py` because it already resides under `opencensus/stats/exporters` so the `_exporter.py` suffix is redundant.

Reply:
Same of the last discussion :smile: